packages/sourcecred/src/cli/update.js

```diff
@@ -0,0 +1,224 @@
+// @flow
+
+import dedent from "../util/dedent";
+import type {Command} from "./command";
+import findLastIndex from "lodash.findlastindex";
+import * as C from "../util/combo";
+import {DiskStorage} from "../core/storage/disk";
+import {loadJsonWithDefault, loadFileWithDefault} from "../util/storage";
+import {join as pathJoin} from "path";
+import stringify from "json-stable-stringify";
+import {encode} from "../core/storage/textEncoding";
+
+/**
+  The `update` CLI can be used by instance maintainers to magically update
+  files in their instance after bumping the sourcecred version. This allows us
+  to be more liberal in making breaking changes, as long as we provide updaters
+  in this module to making the transition super easy for instance maintainers.
+
+  To add an updater, create a new function for it below this string and then
+  register your updater by appending it to the `updates` const defined below.
+ */
+
+const v0_9_0 = async () => {
+  const storage = new DiskStorage(process.cwd());
+  const oldDiscordParser = C.object(
```

Comment:
What are your thoughts on separating out config file log to the appropriate config parser file? I feel like building off of this will generate some significant tech debt, especially when we get around to supporting third party plugins and multiple breaking changes across multiple plugins